packages/grid/_modules_/grid/models/colDef/gridBooleanColDef.tsx

```diff
@@ -4,6 +4,11 @@ import { renderBooleanCell } from '../../components/cell/GridBooleanCell';
 import { renderEditBooleanCell } from '../../components/cell/GridEditBooleanCell';
 import { gridNumberComparer } from '../../utils/sortingUtils';
 import { getGridBooleanOperators } from './gridBooleanOperators';
+import { GridValueFormatterParams } from '../params/gridCellParams';
+
+function gridBooleanFormatter({ value, api }: GridValueFormatterParams) {
+  return api.getLocaleText(value ? 'booleanCellTrueLabel' : 'booleanCellFalseLabel');
```

Comment:
Would it make more sense so we can more easily grep all the usage of getLocaleText?

```suggestion
  return value ? api.getLocaleText('booleanCellTrueLabel') : api.getLocaleText('booleanCellFalseLabel');
```